### ISSUE TITLE
remap error unrecognized argument `-a`

### ISSUE THREAD

**Issue by nate-d-olson** [ACEnglish/truvari#173]
**Version** : 

4.1.0

**Describe the bug** : 

Error message from bwapy, `-a` unrecognized argument.

```
2023-11-06 08:08:00,417 [INFO] Truvari v4.1.0
2023-11-06 08:08:00,417 [INFO] Command /defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/bin/truvari anno remap -r /home/nolson/.cache/snakemake/snakemake/source-cache/runtime-cache/tmph0vdox1x/file/defrabb_runs/code_base/defrabb/rules/../resources/references/GRCh38_chr21.fa -o results/draft_benchmarksets/testG/intermediates/GRCh38_chr21_asm17aChr21_stvar_dipcall-default.remap.vcf --minlength 20 --debug results/draft_benchmarksets/testG/intermediates/GRCh38_chr21_asm17aChr21_stvar_dipcall-default.vcf.gz
Traceback (most recent call last):
  File "/defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/bin/truvari", line 10, in <module>
    sys.exit(main())
  File "/defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/lib/python3.9/site-packages/truvari/__main__.py", line 102, in main
    TOOLS[args.cmd](args.options)
  File "/defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/lib/python3.9/site-packages/truvari/anno.py", line 64, in anno_main
    ANNOS[args.cmd][1](args.options)
  File "/defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/lib/python3.9/site-packages/truvari/annotations/remap.py", line 178, in remap_main
    anno = Remap(in_vcf=args.input,
  File "/defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/lib/python3.9/site-packages/truvari/annotations/remap.py", line 45, in __init__
    self.aligner = BwaAligner(self.reference, options="-a")
  File "/defrabb_runs/code_base/defrabb/.snakemake/conda/8a3ff01a4def5865ecd0d5a4038cdbbf_/lib/python3.9/site-packages/bwapy/libbwa.py", line 210, in __init__
    raise ValueError(
ValueError: Option '-a' is not a valid option (allowed: M Y j k c v s r A B O E U w L d T Q D m N W x G h y X).
```

**To Reproduce** : 
Command used `truvari anno remap             -r /home/nolson/.cache/snakemake/snakemake/source-cache/runtime-cache/tmph0vdox1x/file/defrabb_runs/code_base/defrabb/rules/../resources/references/GRCh38_chr21.fa             -o results/draft_benchmarksets/testG/intermediates/GRCh38_chr21_asm17aChr21_stvar_dipcall-default.remap.vcf             --minlength 20             --debug             results/draft_benchmarksets/testG/intermediates/GRCh38_chr21_asm17aChr21_stvar_dipcall-default.vcf.gz             &> logs/truvari_anno_remap/testG/intermediates/GRCh38_chr21_asm17aChr21_stvar_dipcall-default.log`

**Expected behavior** : 
truvari anno remap should run without error producing annotated vcf.

**Example Data** : 
[GRCh38_chr21_asm17aChr21_stvar_dipcall-default.vcf.gz](https://github.com/ACEnglish/truvari/files/13266985/GRCh38_chr21_asm17aChr21_stvar_dipcall-default.vcf.gz)

URL for GRCh38_chr21.fa used (https://giab-data.s3.amazonaws.com/giab-test-data/GRCh38_chr21.fa.gz) note this is a subset or GRCh38 Chr21 that I use for testing.

**Additional context**:
Specifications file for conda environment used to run `truvari anno remap`
[truvari_env.txt](https://github.com/ACEnglish/truvari/files/13267048/truvari_env.txt)

Note uses `bwapy=0.1.4=py39h3d4b85c_7`

**Comment by ACEnglish** [comment]
I can't figure out what's happening here. There's specifically a bwapy commit that says it adds `-a` support ([link](https://github.com/nanoporetech/bwapy/commit/87a74d4005439b69f290bc2197aba81e5c71024d)). And I can get the file to run on my end. 
```bash
$ bcftools query -f "%POS\t%REMAP\n" -i "REMAP != '.'"  out.vcf
9586586	partial
9676568	interspersed
9719626	interspersed
9719637	tandem
9768070	tandem
9801029	tandem
9882312	interspersed
9981099	interspersed
9997166	interspersed
10009886	interspersed
10014850	interspersed
10129134	partial
```

My best guess is that it is something with the conda installation. Can you try to recreate the issue installing directly from either `pip install bwapy` or building directly from [here](https://github.com/ACEnglish/bwapy). 

Note that bwapy is not maintained anymore by its original authors. I forked it and supported it for a while, but its been giving users problems recently. I don't have the expertise to develop the c-code. I might need to find an alternative to bwapy or deprecate `anno remap`.

**Comment by nate-d-olson** [comment]
Thanks for looking into this. It appears that the `-a` support was added after they released version 0.1.4. As version 0.1.4 is the latest released version, none of the bwapy conda packages support `-a`. I am looking into other ways to get a newer version of the bwapy code into my truvari conda environment. Feel free to close this issue. If I figure anything out I will post a comment incase other truvari users run into the same issue.
